aws_lambda_script/app/resources.py

```diff
@@ -353,6 +353,7 @@ def post(self):
             or "email" not in new_project["user"][0]
             or "name" not in new_project["details"][0]
             or "stage" not in new_project
+            or "supporting_tools" not in new_project
```

Comment:
so supporting_tools needs to be in the payload but each entry within supporting_tools can be empty. I presume that's okay? Meaning it's not mandatory.